src/modules/lua.c

```diff
@@ -1316,6 +1316,32 @@ mtev_lua_open(const char *module_name, void *lmc,
   return L;
 }
 
+int
+mtev_lua_push_inet_ntop(lua_State *L, struct sockaddr *r) {
+  char remote_str[128];
+  int len;
+  switch(r->sa_family) {
+    case AF_INET:
+      len = sizeof(struct sockaddr_in);
+      inet_ntop(AF_INET, &((struct sockaddr_in *)r)->sin_addr,
```

Comment:
Needs `errno` check.

Reply:
this is a pre-existing function... and we can't get an error under this usage condition.,